llvm/test/CodeGen/WebAssembly/bulk-memory.ll

```diff
@@ -1,7 +1,7 @@
-; RUN: llc < %s -asm-verbose=false -verify-machineinstrs -disable-wasm-fallthrough-return-opt -wasm-disable-explicit-locals -wasm-keep-registers -mcpu=mvp -mattr=+bulk-memory | FileCheck %s --check-prefixes CHECK,BULK-MEM
-; RUN: llc < %s -asm-verbose=false -verify-machineinstrs -disable-wasm-fallthrough-return-opt -wasm-disable-explicit-locals -wasm-keep-registers -mcpu=mvp -mattr=-bulk-memory | FileCheck %s --check-prefixes CHECK,NO-BULK-MEM
+; RUN: llc < %s -asm-verbose=false -verify-machineinstrs -disable-wasm-fallthrough-return-opt -wasm-disable-explicit-locals -wasm-keep-registers -mcpu=mvp -mattr=+bulk-memory-opt | FileCheck %s --check-prefixes CHECK,BULK-MEM
```

Comment:
Do we need to replace the current `-mattr=+bulk-memory` with `+bulk-memory-opt` in all the tests where it doesn't matter? I am not strongly opinionated whether we should or should not add the non-Wasm-proposal features, but I'm not sure whether it should be used in all the tests instead of the old features.

Reply:
I'm ok either way. I've now changed bulk-memory.ll and bulk-memory64.ll back to -mattr=+bulk-memory.